examples/examples_test.go

```diff
@@ -7,47 +7,55 @@ import (
 	"path"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
-
 	"github.com/pulumi/pulumi/pkg/testing/integration"
 )
 
-func TestExamples(t *testing.T) {
-	// Ensure we have any required configuration points
-	environ := os.Getenv("ARM_ENVIRONMENT")
-	if environ == "" {
-		t.Skipf("Skipping test due to missing ARM_ENVIRONMENT variable")
+func TestSimple(t *testing.T) {
+	test := getBaseOptions(t).
+		With(integration.ProgramTestOptions{
+			Dir:           path.Join(getCwd(t), "simple"),
+			RunUpdateTest: true,
+			Secrets: map[string]string{
+				"password": "SecretP@sswd99!",
+			},
+		})
+
+	integration.ProgramTest(t, &test)
+}
+
+func skipIfShort(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping long-running test in short mode")
+	}
+}
+
+func getEnviron(t *testing.T) string {
+	env := os.Getenv("ARM_ENVIRONMENT")
+	if env == "" {
+		t.Skipf("Skipping test due to missing ARM_ENVIRONMENT environment variable")
 	}
+
+	return env
+}
+
+func getCwd(t *testing.T) string {
 	cwd, err := os.Getwd()
-	if !assert.NoError(t, err, "expected a valid working directory: %v", err) {
-		return
+	if err != nil {
+		t.Error("expected a valid working directory", err)
 	}
 
-	// base options shared amongst all tests.
-	base := integration.ProgramTestOptions{
+	return cwd
+}
+
+func getBaseOptions(t *testing.T) integration.ProgramTestOptions {
+	environ := getEnviron(t)
+	return integration.ProgramTestOptions{
+		ExpectRefreshChanges: true,
```

Comment:
We didn’t have this previously did we? It would be nice to keep these changes functionally the same if possible.

Reply:
Again, great catch :)